### DISCUSSION TITLE
How can i get remaining three stars?

### DISCUSSION THREAD

**Discussion by reallEz** [ByteLegend/ByteLegend#87] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/85923172/139978546-2e61486f-2515-46e9-875c-bb96e8b7851b.png)

**Comment by gtn1024** [comment, marked as answer]
你点开他，上面应该有「挑战1」「挑战2」相关的字样

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Songbanchai ***@***.***>
Sent: Wednesday, November 3, 2021 9:07:58 AM
To: ByteLegend/ByteLegend ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ByteLegend/ByteLegend] How can i get remaining three stars? (Discussion #87)


[image]<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F85923172%2F139978546-2e61486f-2515-46e9-875c-bb96e8b7851b.png&data=04%7C01%7C%7C76f0c19466d74707942e08d99e66608b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637714984827241253%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=DExn9heM8encKqHxWD1VxyPcECsxsGfqW1GgrNs0f%2BY%3D&reserved=0>

―

**Reply by reallEz** [reply]
Got it.
